hera_pspec/conversions.py

```diff
@@ -60,7 +60,7 @@ class Cosmo_Conversions(object):
     # astropy load attribute
     _astropy = _astropy
 
-    def __init__(self, Om_L=0.68440, Om_b=0.04911, Om_c=0.26442, H0=100.0,
+    def __init__(self, Om_L=0.68440, Om_b=0.04911, Om_c=0.26442, H0=67.27,
```

Comment:
Why was this changed to 67? I think we said we wanted to keep this default at 100.0 such that the output distance units are in h-1 Mpc by default.

Reply:
In isolation, we thought this was ok. But when combined with the way we were calling the conversions module downstream, it meant that the code got very confusing because the default behaviour became one where the results didn't change whether or not you used little_h.

Reply:
okay, so it seems like then we should update the documentation in `conversions.py` to specify that the units don't have little_h in them.

Reply:
The `scalar()` method now has a kwarg called `little_h` that lets the user choose whether to output in h^-1 units. If we don't set H_0 to something other than 100, this parameter has no effect, so I think it makes sense to change the default here.

Reply:
Alright, I just fixed this along the lines that we discussed, where the user can now explicitly ask for little_h to be used or not in every distance function.